README.md

```diff
@@ -0,0 +1,29 @@
+# File Watcher repository contains two projects
+## FileWatcher
+A piece of software designed to look at a instruments lastrun.txt file and determine when a new .nxs file has appeared. Once it has appeared it will send a message to the correct memphis station.
```

Comment:
Can you add a subsection detailing what the lastrun.txt is, where it is, and why it originally exists. I can see this being very confusing for people in the future.

Reply:
It is very confusing for me now, this is a good call.